app/templates/fund_start_page.html

```diff
@@ -66,7 +66,11 @@ <h1 class="govuk-heading-xl">
             <h2 class="govuk-heading-m">{% trans%}Get help {% endtrans %}</h2>
                 {% if link_to_contact_us_page %}
                 <p class="govuk-body">
-                    If you have questions about the form or the fund,
+                    {% if round_short_name == "R4W2" or fund_short_name == "cof-eoi" %}
```

Comment:
@hamzabinkhalid is there any way to move this in to backend code instead of putting a hard coded check ?

Reply:
good catch, moved checks to backend, thanks